src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java

```diff
@@ -30,7 +27,7 @@
      * don't break the main behaviour and result as expected.
      */
     @Test
-    public void testCorrectWorkingExpectedActual() throws IOException {
+    public void testCorrectWorkingExpectedActual() {
```

Comment:
There is no need to prepare all the `test` methods with `test`.

Reply:
why do you think so? As I know, it's based on best practices.

Reply:
You already have the `@Test` annotation. And your class ends with `Test`.

The `test` prefix is an outdated policy that was mandatory for jUnit3. It's not a `best practice`, but if you're so sure, could you please share where such `best practices` are?

Reply:
my bad